src/mainwindow.cpp

```diff
@@ -2753,7 +2753,7 @@ bool MainWindow::eventFilter(QObject *object, QEvent *event)
         auto *keyEvent = static_cast<QKeyEvent *>(event);
         if (keyEvent->key() == Qt::Key_Return && m_searchEdit->text().isEmpty()) {
             setFocusOnText();
-            return true;
+			break;
```

Comment:
Just suggesting to keep the indentation consistent:
```suggestion
            break;
```

Reply:
Alright, thanks.

Reply:
How about we just not put a break in here? There's a `break` after the 'if` condition which will do the same.